### DISCUSSION TITLE
Cloudflare connect using domain

### DISCUSSION THREAD

**Discussion by finnie2006** [AzuraCast/AzuraCast#3938] [Q&A]
Question: I use cloudflare for my azyracast but i want to let connect some radio djs to play on my station i have my own server running on my own network but i cant Connect to mixxx with the domain when cloudflare is enabled is there a way to do that with cloudflare, or is there an free proxy that i can use you guys recommend, or is are there ports cloudflare and icecast support so i can change the port of the mountpoint?

**Comment by BusterNeece** [comment]
@finnie2006 You can just instruct your streamers to connect to the direct IP address of your installation, while listeners can connect using the Cloudflare-powered domain name.
